src/rest-server/src/server.js

```diff
@@ -35,4 +35,11 @@ app.listen(config.serverPort, () => {
   logger.info('RESTful API server starts on port %d', config.serverPort);
 });
 
-module.exports = app;
+esApp.listen(config.esPort, () => {
```

Comment:
I think the elastic search apis should be a sub router of the current rest server so that the third party extensions and rest api users could access these apis easily.

Reply:
Another solution is just like the prometheus server. We just expose the elastic search's endpoint and let user customizes their queries.